katello/katello-restore

```diff
@@ -44,7 +44,9 @@ def restore_databases
     `tar --selinux --overwrite --listed-incremental=/dev/null -xzf pgsql_data.tar.gz -C /`
   else
     `service postgresql start`
+    `runuser - postgres -c "createuser foreman"`
```

Comment:
Possibly loading the dump PR #378 would be sufficient to obsolete this whole PR..... I'm not familiar enough with postgresql to be sure.....